img/forward.svg

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Please clean things up :)
You can use https://jakearchibald.github.io/svgomg/

Or our official scour `scour --create-groups --enable-id-stripping --enable-comment-stripping --shorten-ids --remove-metadata --strip-xml-prolog --no-line-breaks img/forward.svg img/forward.new.svg`